examples/rust/README.md

```diff
@@ -0,0 +1,40 @@
+# Examples of Zenoh Rust applications communicating with ROS 2 Nodes
+
+## Building the examples
+In order to build the examples you will need to:
+* [Install Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
+* [Install Zenoh Rust](https://github.com/eclipse-zenoh/zenoh?tab=readme-ov-file#how-to-install-it)
```

Comment:
No need to install Zenoh Rust. As it's a dependency declared in `Cargo.toml`, cargo will automatically download and build it.